docs: adds configuration template to revert standard naming convention for Azure Pipelines (#20208)
renovatebot/renovate: lib/modules/manager/azure-pipelines/readme.md

}
```

In most cases only major version numbers are specified in YAML when referencing a task version, e.g. `NodeTool@0`. Renovate, by default, will replace these with the full version, e.g. `NodeTool@0.216.0`.

To use the standard convention for Azure Pipelines, add:

```json
{
  "packageRules": [
    {
      "matchDatasources": ["azure-pipelines-tasks"],
      "extractVersion": "^(?<version>\\d+)"
    }
  ]
}
```

> It updates container and repository resources from the `resources` block as well as tasks from `steps` blocks.

Reviewer: ```suggestion
Renovate now updates container and repository resources from the `resources` block, plus tasks from `steps` blocks.
```
